zebra-consensus/src/checkpoint.rs

```diff
@@ -311,9 +311,7 @@ where
         if matches!(howudoin::cancelled(), Some(true)) {
             self.finish_diagnostics();
         } else {
-            self.queued_blocks_bar
-                .set_pos(max_queued_height.0)
-                .set_len(u64::from(self.checkpoint_list.max_height().0));
+            self.queued_blocks_bar.set_pos(max_queued_height.0);
```

Comment:
The queued blocks progress bar is guaranteed to "complete". As long as Zebra is connected to the network, it will queue the final checkpoint, verify it, then shut down this progress bar.

Reply:
Restored in 74365c42a2ac646690d6d7c0e1fdc840d5aca043